Formula/clozure-cl.rb

```diff
@@ -0,0 +1,46 @@
+class ClozureCl < Formula
+  desc "Common Lisp implementation with a long history"
+  homepage "https://ccl.clozure.com"
+  url "https://github.com/Clozure/ccl/archive/v1.11.5.tar.gz"
+  sha256 "07c7e05c1d50ccbf1f7602a1d436b6d6da5dcda7656b89b74daa4cf833fc7929"
+  head "https://github.com/Clozure/ccl.git"
+
+  depends_on :xcode => :build
+
+  conflicts_with "cclive", :because => "both install a ccl binary"
+
+  resource "bootstrap" do
+    url "https://github.com/Clozure/ccl/releases/download/v1.11.5/ccl-1.11.5-darwinx86.tar.gz"
+    sha256 "5adbea3d8b4a2e29af30d141f781c6613844f468c0ccfa11bae908c3e9641939"
+  end
+
+  def install
+    tmpdir = Pathname.new(Dir.mktmpdir)
+    tmpdir.install resource("bootstrap")
+    buildpath.install tmpdir/"dx86cl64.image"
+    buildpath.install tmpdir/"darwin-x86-headers64"
+    cd "lisp-kernel/darwinx8664" do
+      system "make"
+    end
+
+    ENV["CCL_DEFAULT_DIRECTORY"] = buildpath
+
+    system "./dx86cl64", "-n",
+           "-l", "lib/x8664env.lisp",
```

Comment:
Align next lines with the `"-n"`, as I did in the code I pasted

Reply:
@fxcoudert I don't quite understand what you mean here. Could you please give an example?

Reply:
@fxcoudert Oh, ok, I didn't know which paste you were referring to. I committed that just now.